playground/DatabaseMigration/README.md

```diff
@@ -0,0 +1,10 @@
+# Data migrations playground
+
+To use migrations:
```

Comment:
What happens when you apply migrations with the command line and this approach? Does it throw still? Just later?

Reply:
Do you mean what happens if you call `dotnet ef database update`? It will error and say there is no connection string. You can specify a `--connectionstring` argument.

Maybe this scenario would be interesting once Aspire has better support for persistent containers + DB, and you can copy the connection string and include it with the command. It seems annoying to use compared to applying migrations at app startup.

Reply:
That's a good point. To make this work with the command line you'd need:
1. The connection string in the project
2. The db container needs to be running.

This feels like good incremental progress.